benchexec/tablegenerator/react-table/package.json

```diff
@@ -69,7 +69,9 @@
     "modulePaths": [],
     "moduleNameMapper": {
       "^react-native$": "react-native-web",
-      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
+      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy",
+      "imports-loader?.*": "<rootDir>/src/tests/mocks/importsLoader.js",
```

Comment:
You said that tests were breaking because of `imports-loader`, but in 9137bf4 I specifically added the line `"^.*/src/vendor/.*$": "<rootDir>/config/jest/fileTransform.js"` above to keep them working, and they do work for me and in the CI.

In which circumstances does it not work?

Reply:
I cleaned my environment and forced a npm reinstall. After that it worked. 
I think something was ccorrupted on my end for whatever reason. :/